plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1363,6 +1485,17 @@ void producer_plugin_impl::plugin_startup() {
             }
          }));
 
+         if (!_producers.empty()) { // track votes if producer to verify votes are being processed
+            auto on_vote_signal = [this]( const vote_signal_params& vote_signal ) {
+               try {
+                  const auto& [connection_id, status, msg, active_auth, pending_auth] = vote_signal;
+                  on_vote(connection_id, status, msg, active_auth, pending_auth);
```

Comment:
No big deal, but why not have `on_vote` take `const vote_signal_params& ` and unbind in `on_vote`?

Reply:
It separates concerns of the signaled type/info from what `on_vote` needs.